PyStemmusScope/forcing_io.py

```diff
@@ -0,0 +1,138 @@
+import os
+import numpy as np
+import xarray as xr
+
+
+def _calculate_ea(t_air_celcius, rh):
```

Comment:
Can we make a separate module for this function and import it here? For example, we can create `ancillary_vars.py `or `derive_vars.py` for these types of functions.